src/Utilities/CssConcatenator/RuleBlock.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Utilities\CssConcatenator;
+
+/**
+ * Implementation of media rules sub-property for \Pelago\Emogrifier\Utilities\CssConcatenator
+ *
+ * @internal
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ * @author SignpostMarv
+ */
+class RuleBlock
+{
+    /**
+     * Array whose keys are selectors for the rule block (values are of no
+     *     significance);
+     *
+     * @var mixed[]
+     *
+     * @psalm-var array<string, mixed>
+     */
+    public $selectorsAsKeys = [];
```

Comment:
if the values are of no significance, should this be adjusted to `array<string, int>` as implemented, or `array<string, null>` to avoid the temptation to add random ints with special purpose?

Reply:
> if the values are of no significance

That's right, they aren't.  From memory they might be `bool`s (that's the usual construct in this situation).  But I'm still yet to review this PR as a whole...

Reply:
at least one of them is a zero.